pr2eus_tutorials/launch/pr2_tabletop.launch

```diff
@@ -2,14 +2,31 @@
   <arg name="run_rviz" default="false" />
   <arg name="publish_objects_tf" default="true" />
   <arg name="launch_tracking" default="false" />
-  <arg name="manager" default="pr2_tabletop_object_detector_nodelet_manager"/>
-  <arg name="machine" default="c2" />
-  <arg name="load_machine" default="false" />
+  <arg name="manager" default="pr2_tabletop_object_detector_nodelet_manager" />
+  <arg name="remote" default="false" />
+  <arg name="load_machine" default="false" unless="$(arg remote)" />
+  <arg name="load_machine" default="true" if="$(arg remote)" />
+  <arg name="machine" default="c2" unless="$(arg remote)" />
+  <arg name="machine" default="localhost" if="$(arg remote)" />
+  <arg name="camera_name" default="kinect_head" />
+
+  <arg name="RGB_CAMERA_INFO" value="/$(arg camera_name)/rgb/camera_info" />
+  <arg name="RGB_IMAGE" value="/$(arg camera_name)/rgb/throttled/image_rect_color" />
+  <arg name="RGB_IMAGE_REMOTE" value="/$(arg camera_name)_remote/rgb/image_rect_color" />
+  <arg name="DEPTH_CAMERA_INFO" value="/$(arg camera_name)/depth_registered/camera_info" />
+  <arg name="DEPTH_IMAGE" value="/$(arg camera_name)/depth_registered/image_rect" />
+  <arg name="DEPTH_IMAGE_REMOTE" value="/$(arg camera_name)_remote/depth_registered/image_rect" />
+  <arg name="PROSILICA_CAMERA_INFO" value="/prosilica/camera_info" />
+  <arg name="PROSILICA_IMAGE" value="/prosilica/image_raw" /> 
+  <arg name="PROSILICA_IMAGE_REMOTE" value="/prosilica_remote/image_raw" /> 
+  <arg name="POINT_CLOUD" value="/$(arg camera_name)/depth_registered/throttled/points" />
+  <arg name="POINT_CLOUD_REMOTE" value="/$(arg camera_name)_remote/depth_registered/points" />
```

Comment:
```
<arg name="DEPTH_IMAGE" value="/$(arg camera_name)/depth_registered/image_rect" />
```
should be

```
<arg name="DEPTH_IMAGE" value="/$(arg camera_name)/depth_registered/throttled/image_rect" />
```
?

Reply:
Yes, thank you.
Fix by https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/459/commits/bc631385ac9c173861f6ebba637fba173e09ba34 .